packages/ui/src/components/Main/Search.tsx

```
@@ -0,0 +1,266 @@
+        <summary>Search results ({list?.length ?? 0} results)</summary>
+        <Flex direction={'column'} gap={'1'}>
+          {list?.map((item: SearchResultItem) => (
+            <Card size={'1'} key={item.id}>
+              <Flex direction={'row'} gap={'2'} align={'center'}>
+                <Popover.Root>
+                  <Popover.Trigger>
+                    <Avatar src={item.thumbnail} fallback={<VideoIcon />} size={'4'} />
+                  </Popover.Trigger>
+                  <Popover.Content>
+                    <img src={item.thumbnail} width={640} />
+                  </Popover.Content>
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)